src/arrayMethodSort.js

```diff
@@ -5,7 +5,22 @@
  */
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+    const callback = compareFunction
+      || ((prevElem, elem) => elem.toString() < prevElem.toString());
```

Comment:
Some values don't have the toString method, consider rewriting to string coercion